java/code/webapp/WEB-INF/nav/system_detail.xml

```diff
@@ -185,7 +185,7 @@
     <rhn-tab name="Join" url="/rhn/systems/details/groups/Add.do"/>
   </rhn-tab>
 
-  <rhn-tab name="Virtualization" acl="not system_is_virtual(); system_has_management_entitlement() or system_has_salt_entitlement()">
+  <rhn-tab name="Virtualization" acl="system_has_virtualization_entitlement()">
```

Comment:
Where this entitlement has been defined?

Reply:
Defined for years, but never used:
https://github.com/uyuni-project/uyuni/blame/master/java/code/src/com/redhat/rhn/common/security/acl/Access.java#L174